### PR TITLE
Using printf for custom and non custom formats

### PR DESCRIPTION
**Pull request by fredmann1993**

### CODE REVIEW

File: 1-conv_specifiers.c [hunk 1]
```diff
@@ -45,15 +45,15 @@ int print_unsigned(va_list l, flags_t *f)
...
 	{
 		_putchar('-');
-		n1 = -n;
+		num1 = -n;
 	}
 	else
-		n1 = n;
+		num1 = n;
 
 	if (n1 / 10)
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: you should change it here too

File: 1-conv_specifiers.c [hunk 2]
```diff
@@ -45,15 +45,15 @@ int print_unsigned(va_list l, flags_t *f)
...
 		print_number(n1 / 10);
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: Here as well

File: 1-conv_specifiers.c [hunk 3]
```diff
@@ -68,7 +68,7 @@ void print_number(int n)
  */
 int count_digit(int i)
 {
-	unsigned int d = 0;
+	unsigned int dig = 0;
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: I think "d" is quite good. Please can we change it back to "d" .

File: 1-conv_specifiers.c [hunk 4]
```diff
@@ -10,14 +10,14 @@
 int print_int(va_list l, flags_t *f)
 {
 	int n = va_arg(l, int);
-	int res = count_digit(n);
+	int result = count_digit(n);
 
 	if (f->space == 1 && f->plus == 0 && n >= 0)
 		res += _putchar(' ');
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: change it here too

File: 1-conv_specifiers.c [hunk 5]
```diff
@@ -10,14 +10,14 @@
...
 	if (f->plus == 1 && n >= 0)
 		res += _putchar('+');
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: Here as well

File: 1-conv_specifiers.c [hunk 6]
```diff
@@ -10,14 +10,14 @@
...
 	if (n <= 0)
-		res++;
+		result++;
 	print_number(n);
 	return (res);
```
Comment (betiniakarandut) [inline 1-conv_specifiers.c]: Here as well

**Review by betiniakarandut — COMMENTED** [review]
Okay. Please check again and make the necessay changes/additions

File: 1-conv_specifiers.c (hunk 1 above)
Comment (fredmann1993) [reply 1-conv_specifiers.c]: ok

File: 1-conv_specifiers.c (hunk 6 above)
Comment (fredmann1993) [reply 1-conv_specifiers.c]: I have updated the variable names

**Pull request closed without merging**